tests/pr_test.go

```diff
@@ -54,6 +55,22 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
```

Comment:
So in order to test the advanced example, lets copy the approach [here](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/0295160a963d3beb60394bfd60da6a42513f83f1/tests/pr_test.go#L87-L127) which will deploy the example in schematics which means the private endpoint can be used.